java-alternatives/run.sh

```diff
@@ -56,3 +56,10 @@ set_java_alternatives() {
 }
 
 set_java_alternatives
+
+SCRIPT_DIR=$(dirname $0)
+ADDED_DIR=${SCRIPT_DIR}/added
+
+mkdir -p ${JBOSS_HOME}/bin/launch
+cp -r ${ADDED_DIR}/launch/* ${JBOSS_HOME}/bin/launch
+
```

Comment:
Would it not be better to include this within the os-java-run scripts so it is available to all java images?